es6/tests/base.js

```diff
@@ -924,3 +926,44 @@ describe("Serialization", function() {
 		JSON.stringify(doc);
 	});
 });
+
+describe("Docxtemplater v4 tests", function() {
+	it("should work with new constructor", function() {
+		let isModuleCalled = false;
+
+		const module = {
+			optionsTransformer(options) {
+				isModuleCalled = true;
+				return options;
+			},
+		};
+
+		createDocV4("tag-example.docx", { modules: [module] });
+		expect(isModuleCalled).to.equal(true);
+	});
+
+	it("should throw error when modules passed is not an array", function() {
+		expect(createDocV4.bind(this, "tag-example.docx", { modules: {} })).to.throw(
+			"The modules argument of docxtemplater's constructor must be an array"
+		);
+	});
+
+	it("should throw error when a valid zip is not passed", function() {
```

Comment:
it("should throw error when a valid zip is not passed", function() {

=>

	it("should throw error when an invalid zip is passed", function() {

Reply:
Yup will do it.